### PR TITLE
Feature/as 19 lpa questionnaire development plan

### PR DESCRIPTION
**Pull request by swdpcomputing**
## Ticket Number

AS-19

## Description of change
add development page

## Checklist

- [ ] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [x] My commit history in this PR is linear
- [x] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `master` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: lpa-submissions-e2e-tests/cypress.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "demoDelay": 0
   },
   "retries": {
-    "runMode": 1,
+    "runMode": 0,
```
Comment (davehlewis) [inline lpa-submissions-e2e-tests/cypress.json:11]: Is this change mean to be in this PR?

File: lpa-submissions-e2e-tests/cypress/support/appeals-development-plan/developmentPlanRadioButton.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+/// <reference types = "Cypress"/>
```
Comment (a6software) [inline lpa-submissions-e2e-tests/cypress/support/appeals-development-plan/developmentPlanRadioButton.js:1]: :+1: but maybe globally ? 

https://docs.cypress.io/guides/tooling/intelligent-code-completion.html#Reference-type-declarations-via-jsconfig

File: lpa-submissions-e2e-tests/cypress/support/commands.js [hunk 3]
```diff
@@ -252,3 +246,33 @@ Cypress.Commands.add(
...
+
+Cypress.Commands.add(
+  'developmentPlanRadioButton',
+  require('./appeals-development-plan/developmentPlanRadioButton'),
+);
+
+Cypress.Commands.add(
+  'inputDevelopmentPlanDetails',
+  require('./appeals-development-plan/inputDevelopmentPlanDetails'),
+);
```
Comment (a6software) [inline lpa-submissions-e2e-tests/cypress/support/commands.js:278]: Before this file gets out of hand (as it has done for FWA), it maybe worth considering extracting here:

https://github.com/Planning-Inspectorate/appeal-planning-decision/blob/master/e2e-tests/cypress/support/commands.js

Not essential by any means, just a thought / suggestion.

Maybe overkill also, idk anything about this side of the project.

**Review by a6software — COMMENTED** [review]
LGTM - will approve once @davehlewis comment been answered

File: lpa-submissions-e2e-tests/cypress.json (hunk 1 above)
Comment (swdpcomputing) [reply lpa-submissions-e2e-tests/cypress.json]: Reverted

**Review by davehlewis — APPROVED** [review]
LGTM

**Review by a6software — APPROVED** [review]

File: lpa-submissions-e2e-tests/cypress/support/appeals-development-plan/developmentPlanRadioButton.js (hunk 2 above)
Comment (swdpcomputing) [reply lpa-submissions-e2e-tests/cypress/support/appeals-development-plan/developmentPlanRadioButton.js:1]: @shilpabasu22 Hi Shilpa can you advise? This is based on your style.

File: lpa-submissions-e2e-tests/cypress/support/commands.js (hunk 3 above)
Comment (swdpcomputing) [reply lpa-submissions-e2e-tests/cypress/support/commands.js:278]: @shilpabasu22 Hi Shilpa can you advise? This is based on your style.

Comment (shilpabasu22) [reply lpa-submissions-e2e-tests/cypress/support/commands.js:278]: @swdpcomputing I think Team 1 had hit the issue where command.js file had grown very big. We can edit/update our command.js file to mimic what Team1 has implemented. If you want, I can do that separate to this ticket as it doesn't impact the functionality that you have been working on.

Comment (swdpcomputing) [reply lpa-submissions-e2e-tests/cypress/support/commands.js:278]: Sounds good and sensible. @a6software You happy with this?

File: lpa-submissions-e2e-tests/cypress/support/appeals-development-plan/developmentPlanRadioButton.js (hunk 2 above)
Comment (shilpabasu22) [reply lpa-submissions-e2e-tests/cypress/support/appeals-development-plan/developmentPlanRadioButton.js:1]: again we can implement this. Again this is not a blocker for your ticket and I can do this as part of the change that I will do for command.js file

Comment (swdpcomputing) [reply lpa-submissions-e2e-tests/cypress/support/appeals-development-plan/developmentPlanRadioButton.js:1]: Sounds good and sensible. @a6software You happy with this?

File: lpa-submissions-e2e-tests/cypress/support/commands.js (hunk 3 above)
Comment (a6software) [reply lpa-submissions-e2e-tests/cypress/support/commands.js:278]: Yes, absolutely - both were only suggestions, all good from my pov. Cheers both!

**Pull request merged**